specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/namespaces-preview.json

```diff
@@ -663,7 +663,8 @@
             },
             "zoneRedundant": {
               "type": "boolean",
-              "description": "Enabling this property creates a Standard Event Hubs Namespace in regions supported availability zones."
+              "description": "Enabling this property creates a Standard Event Hubs Namespace in regions supported availability zones.",
+              "default": false
```

Comment:
Let's remove default value for zoneRedundant, it is going to be true by default in the days to come.